autoextract/__main__.py

```diff
@@ -24,7 +24,7 @@
 
 
 async def run(query: Query, out, n_conn, batch_size, stop_on_errors=False,
-              api_key=None):
+              api_key=None, max_query_error_retries=0):
```

Comment:
Since it's already a default value in the CLI, it could be mandatory here.
Unless, the `run` interface needs to have it as optional

Reply:
I guess it's good for communicating that this is an optional parameter.